### ISSUE TITLE
Nose Steering not Really Working well

### ISSUE THREAD

**Issue by EuropeanVirtual** [headwindsim/A339X#52]
### Description

If i will Taxi i can't do make a Right or Left turn with 10 Knots or 20 Knots. Thats really Sad i've to Brake down to 1 or 2 Knots to make it  

### Additional info (optional)

I'm not the Only one Person how have this Problem. 

### Discord Username (optional)

Rowside#7166

**Comment by Revyn112** [comment]
Make sure to setup the nose wheel steering correctly:
https://docs.flybywiresim.com/fbw-a32nx/feature-guides/nw-tiller/

**Comment by EuropeanVirtual** [comment]
> Make sure to setup the nose wheel steering correctly: https://docs.flybywiresim.com/fbw-a32nx/feature-guides/nw-tiller/



the thing is on the A2nx is all Working Fine  no Problems

**Comment by arl1004** [comment]
I had difficulty with the nose wheel steering the last few days and believe there was a setting which wasn't "remembered" perhaps from the config in the EFB following some updates recently. I felt as though the nose wheel wasn't reaching its full turning angle and I was missing turns, struggling to get onto stand etc and so after reading your comments this morning I took the following steps.

1. Checked my profile set up in Control Options and verified my A339 profile for my joystick had the Nose Wheel Steering Axis set up correctly. I tested this and saved the profile again. It was set up correctly.
2. Via the EFB, I disabled the "Separate Tiller from Rudder Inputs" option and enabled again so that my Nose Wheel Steering Axis option would work correctly as I use rudder pedals.

On taxi, I found my nosewheel steering was significantly improved and also visually the nose wheel turned at a greater angle, so whilst this may work OK on the A32NX presently for you, I'd recommend running through the guide again and definitely check the EFB option to almost, "re set up", just in case, as my taxi out in SBGL was perfect this morning.

**Comment by FlyingFoxVATSIM** [comment]
I had the same issue and by doing what **arl1004** described, it did not fully resolve the issue. It works very well on the FBW A320, but only mediocre on the HWD A339.

**Comment by cptbigjay** [comment]
I also have been experiencing this for quite some time on both the latest Main and Development versions, nothing I do helps the problem.

**Comment by FlyingFoxVATSIM** [comment]
> I also have been experiencing this for quite some time on both the latest Main and Development versions, nothing I do helps the problem.
What helps me most of the time is taxiing reaaaaaaaally slow when I want to mark sharp turns. But it is not consistent, it is as if something was working against my input. I have checked all axes to make sure that nothing interferes. No joy.

I hope that the developers can visit this topic and review it.

**Comment by Monkyofwar** [comment]
Hi All,

We are trying are hardest to perfect this in are experimental builds, please bare with us.
This issue is mostly buffed out now.

**Current way to fix.**
Taxi really slow and don't full turn your rudder.

Regards,
Reggie
**Headwind Community Management**

**Comment by FlyingFoxVATSIM** [comment]
> **Current way to fix.**
> Taxi really slow and don't full turn your rudder.

Hi Reggie!

Thanks for replying and taking care of this issue!
I am not using my rudder pedals to steer the aircraft on the ground, but I have a separate axis that I use for nosewheel steering only. Still, the nosewheel will not go/stay at its extreme deflection.
I hope a solution can be found eventually :)
All the best, Andreas

**Comment by jlemonz** [comment]
Yes hope this get fixed. Its really hard to taxi. You see the nose gear just glide over the ground. Only when i go really slow and with brakes i get it to make a turn. I did check the docs. Besides that it is a really cool plane.

**Comment by masterrob94** [comment]
I don't know if it is worth looking at

https://github.com/flybywiresim/aircraft/commit/08a1ef8f5540dfb3fefc4fb8a9487c6526fd06fb

Bigbus A380 does it slightly differently
